policies/submissions.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+title: Submitting Your Project To CODEVID-19
+---
+
+Read these carefully, otherwise we can't consider your team for prizing!
+
+### Deliverables
+
+1. REQUIRED: A link to your working app (whether that's a FindCollabs link, the website URL, or a TestFlight/Google Play opt-in URL).
+2. REQUIRED: A link to a screenshot we can share with voters.
+3. REQUIRED: Instructions for interacting with your app (if not included in the app itself).
+4. REQUIRED: An email address that judges can use to follow up with the team should they have questions or concerns. This will not be shared with the public, or with voters.
+5. REQUIRED: A link to your project on FindCollabs
+6. OPTIONAL: For open-source projects, we encourage a link to the source code repository (e.g. GitHub, GitLab, Bitbucket, etc.).
+
+### General Rules
+
+*   Your solution does not need to be open-source, but it must be available (upon request) for scrutiny by organizers and judges.
+*   In the spirit of hackathons, only apps that were started **on or after March 16, 00:00 UTC** will be considered for prizing.
```

Comment:
Should this be added to the fine print?  Hmm maybe not.  There's a story to add a terms of use, which makes the fine print look quite small (and the Ontario Bar Assoc can help us draft)

Reply:
Let's leave this were it is for now. It's explained in more detail in the `rules.md` so we're just reminding teams here. 

I'll add a link to the full rules in this section just in case teams have missed reading them in getting here.

We can do another pass after we've gotten feedback from the Bar Association but in all fairness, that will likely take them a while to do as they've probably got other priorities right now.

Reply:
I've cleaned up this whole section and clarified that people need to read the full rules. I just mentioned that you don't need to be open source but that it's encouraged to be.